apps/src/lab2/views/dialogs/ShareDialog.tsx

```diff
@@ -190,15 +190,16 @@ const ShareDialog: React.FunctionComponent<{
   const theme = Lab2Registry.getInstance().getTheme();
 
   return sharingDisabled() ? (
-    <Dialog
-      title={i18n.sharingDisabledTitle()}
-      description={i18n.sharingBlockedByTeacherOpenEndedProjects()}
-      mode={theme === 'Light' ? 'light' : 'dark'}
-      primaryButtonProps={{
-        onClick: () => dispatch(hideShareDialog()),
-        text: i18n.ok(),
-      }}
-    />
+    <div data-theme={theme}>
```

Comment:
I updated to use `Modal` instead of `Dialog` since `Modal` has been recently updated to be themed.